app/src/main/java/com/vietdung/sqliteandcontentprovider/ContactAdapter.java

```diff
@@ -0,0 +1,51 @@
+package com.vietdung.sqliteandcontentprovider;
+
+import android.content.Context;
+import android.support.annotation.NonNull;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+
+import java.util.List;
+
+public class ContactAdapter extends RecyclerView.Adapter<ContactAdapter.ViewHolder> {
+    private List<Contact> mContacts;
+    private Context mContext;
+
+    public ContactAdapter(List<Contact> contacts, Context context) {
```

Comment:
convention 2.9
https://github.com/framgia/coding-standards/blob/master/eng/android/codingstyleguide.md#29-parameter-ordering-in-methods